addons/l10n_it_edi/models/account_invoice.py

```diff
@@ -37,8 +37,11 @@ def _get_l10n_it_amount_split_payment(self):
         amount = 0.0
         if self.is_invoice(True):
             for line in [line for line in self.line_ids if line.tax_line_id]:
-                if line.tax_line_id._l10n_it_is_split_payment() and line.credit > 0.0:
-                    amount += line.credit
+                if line.tax_line_id._l10n_it_is_split_payment():
+                    if self.move_type == 'out_invoice' and line.credit > 0.0:
```

Comment:
```suggestion
                    if self.move_type == 'out_invoice':
```
Why do we need to check `credit`, I don't think it can be a negative value? :thinking:

Reply:
You're probably right, since we check the tax that has the VE38 tag, and it should already be the positive one. To be verified though.
![immagine](https://github.com/user-attachments/assets/5d1c3909-b54c-463e-808c-63547b20d561)

Reply:
Yes. It's the positive one.

I forgot to handle the case of the "out_receipt" type.
Should it include the credit or the debit?